### ISSUE TITLE
Keyboard navigation doesn't work

### ISSUE THREAD

**Issue by demonfiddler** [onesine/react-tailwindcss-select#71]
When the component has the focus (which BTW by default isn't highlighted with any visual focus effect and unclear how to achieve this), keyboard navigation does not work as expected:
- when collapsed: spacebar, enter, down, alt+down, ctrl+down: none of these keys activates the drop-down menu
- when expanded:
  - up, down does not change the selection
  - enter does not choose the select value and close the drop-down
  - esc does not close the drop-down

Sadly, for me, this means that this otherwise interesting component is not usable in its present state.
